### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by stevei101**
- changes specific for open-jdk-8
- add example playbook and inventory

### CODE REVIEW

File: playbook.example [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+- hosts: hadoop.master, hadoop.core, hadoop.task
+  become: True
+  roles:
+    - role: geerlingguy.java
```
Comment (micafer) [inline playbook.example:5]: We prefer that this role do not depend on external roles.

File: playbook.example [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      when: "ansible_os_family == 'RedHat'"
+      java_packages:
+        - java-1.8.0-openjdk
+      java_home:
+        - '/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.121-0.b13.el7_3.x86_64/jre'
+
+- hosts: hadoop.master
+  become: True
+  roles:
+      - { role: 'ostrichops.hadoop', hadoop_master: 'MASTER_NODE_NAME_OR_IP', hadoop_type_of_node: 'master', environment.JAVA_HOME: '/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.121-0.b13.el7_3.x86_64/jre'}
```
Comment (micafer) [inline playbook.example:15]: The rolename must be indigo-dc.hadoop.

File: playbook.example [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+- hosts: hadoop.core, hadoop.task
+  become: True
+  roles:
+      - { role: 'ostrichops.hadoop', hadoop_master: 'MASTER_NODE_NAME_OR_IP'}
```
Comment (micafer) [inline playbook.example:20]: Same here.

File: tasks/install.yml [hunk 4]
```diff
@@ -31,8 +30,8 @@
...
-- lineinfile: dest="{{ hadoop_home }}/etc/hadoop/hadoop-env.sh" regexp=JAVA_HOME= line="export JAVA_HOME={{item}}"
-  with_first_found:
-   - /usr/lib/jvm/java-8-openjdk-amd64
-   - /usr/lib/jvm/java-7-openjdk-amd64
-   - /usr/lib/jvm/jre-1.7.0
+- lineinfile:
+    dest: "{{ hadoop_home }}/etc/hadoop/hadoop-env.sh"
+    regexp: '^JAVA_HOME='
+    line: 'JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.121-0.b13.el7_3.x86_64/jre'
+
```
Comment (micafer) [inline tasks/install.yml:37]: Ubuntu 14 does not have packages for java 8, so we need to maintain the "with_first_found" to assure it works in it.

**Review by micafer — CHANGES_REQUESTED** [review]

**Comment by stevei101** [comment]
sorry this was submitted by mistake

**Pull request closed without merging**